haxepunk/graphics/BitmapText.hx

```diff
@@ -240,8 +214,7 @@ class BitmapText extends Graphic
 			parseText();
 
 			textWidth = textHeight = 0;
-			if (blit) _bufferDirty = true;
-			else computeTextSize();
+			computeTextSize();
```

Comment:
I'll push a change to this branch soon. We can now get rid of `computeTextSize()` (text size will be known after running `parseText`) and remove the `renderFonts` function, moving all the rendering logic directly into `render`.